pkg/crc/constants/constants_linux.go

```diff
@@ -1,7 +1,16 @@
 package constants
 
+import (
+	"path/filepath"
+
+	"github.com/code-ready/gvisor-tap-vsock/pkg/transport"
+)
+
 const (
 	OcExecutableName          = "oc"
 	PodmanExecutableName      = "podman"
 	AdminHelperExecutableName = "admin-helper-linux"
+	TapSocketPath             = transport.DefaultURL
```

Comment:
`URL` or `socketPath` ? I think the name is vsock is too generic and, and should actually be something like:

`vsockResource` or `vsockPath`... as it is not an actual url, but rather an URI

Reply:
I simplified this.